docs/src/man/comparisons.md

```diff
@@ -1,77 +1,204 @@
+
+| Operation                | pandas                                                | DataFrames.jl                           |
+| :----------------------- | :---------------------------------------------------- | :-------------------------------------- |
+| Reduce multiple values   | `df['z'].mean(skipna = False)`                        | `mean(df.z)`                            |
+|                          | `df['z'].mean()`                                      | `mean(skipmissing(df.z))`               |
+|                          | `df[['z']].agg(['mean'])`                             | `combine(df, :z => mean ∘ skipmissing)` |
+| Add new columns          | `df.assign(x_mean = df['x'].mean())`                  | `transform(df, :x => mean => :x_mean)`  |
+| Rename columns           | `df.rename(columns = {'x': 'x_new'})`                 | `rename(df, :x => :x_new)`              |
+| Pick & transform columns | `df.assign(x_mean = df['x'].mean())[['x_mean', 'y']]` | `select(df, :x => mean, :y)`            |
+| Sort rows                | `df.sort_values(by = ['x'])`                          | `sort(df, :x)`                          |
+
+Note that Julia propagates `missing` data by default for safety reasons. The `skipmissing` function
+can be used to remove missing data. See more details at the [Additional Differences](@ref) section below.
+
+### Grouping data and aggregation
+
+| Operation                    | pandas                                                                                    | DataFrames.jl                               |
+| :--------------------------- | :---------------------------------------------------------------------------------------- | :------------------------------------------ |
+| Aggregate by groups          | `df.groupby('grp')['x'].mean().reset_index()`                                             | `combine(groupby(df, :grp), :x => mean)`    |
+| Aggregate and add columns    | `df.join(df.groupby('grp')['x'].mean(), on='grp', rsuffix='_mean')`                       | `transform(groupby(df, :grp), :x => mean)`  |
+| Aggregate and select columns | `df.join(df.groupby('grp')['x'].mean(), on='grp', rsuffix='_mean')[['grp','x_mean','y']]` | `select(groupby(df, :grp), :x => mean, :y)` |
+
+### More advanced commands
+
+| Operation                 | pandas                                                                       | DataFrames.jl                                                 |
+| :------------------------ | :--------------------------------------------------------------------------- | :------------------------------------------------------------ |
+| Complex Function          | `df[['z']].agg(lambda v: np.mean(np.cos(v)))`                                | `combine(df, :z => v -> mean(cos, skipmissing(v)))`           |
+| Transform several columns | `df.agg({'x': max, 'y': min})`                                               | `combine(df, :x => maximum, :y => minimum)`                   |
```

Comment:
same here, the equivalent of what you write in pandas now in DataFrames.jl would be `[maximum(df.x), minimum(df.y)]`.

Reply:
Actually, `[maximum(df.x), minimum(df.y)]` isn't the same because the information about the column names are missing in the result. In pandas, the row index contains the column being aggregated:

```
>>> df.agg({'x': max, 'y': min})
x    6
y    4
dtype: int64
```